test-integration/helpers/scores.coffee

```diff
@@ -50,11 +50,17 @@ class ScoresHelper extends TestHelper
     @el.ccScoresLink.click()
     @waitUntilLoaded()
 
-  doneGenerating: =>
-    #@test.driver.wait =>
-      #@test.driver.isElementPresent(css: @el.downloadExport)
+  downloadExport: =>
+    #@test.utils.wait.giveTime 10000, =>
+    @test.sleep(5000, 'waiting for export')
+    # Selenium does not interact with elements that have display: none set
+    @test.driver.executeScript("document.getElementById('downloadExport').style.display='block';")
+    @el.exportUrl.get().getAttribute("src").then (src) ->
```

Comment:
did you do the `style.display='block'` so that `.get` can grab it?  does taking out line 57 and using `findElement` instead of `get` work?